src/effects/effectchainslot.h

```diff
@@ -133,11 +135,24 @@ class EffectChainSlot : public QObject {
     ControlPushButton* m_pControlChainEnabled;
     ControlObject* m_pControlChainMix;
     ControlObject* m_pControlChainSuperParameter;
+    ControlObject* m_pControlChainSuperForceUpdate;
     ControlPushButton* m_pControlChainInsertionType;
     ControlObject* m_pControlChainSelector;
     ControlPushButton* m_pControlChainNextPreset;
     ControlPushButton* m_pControlChainPrevPreset;
 
+    /**
```

Comment:
I go with @Be-ing here, since even if we implement an other solution later the interface can likely be the same. 

The outlined solution will finally work, but it introduces a lot more code to maintain, and the original chicken-egg problem is still there: Who controls the lifetime of the CO the skin or the Controller.

We have solved a quite similar problem with the view menu recently. In that case the ownership was clear. The skin is the owner, and requests Mixxx to show the view controls.